### ISSUE TITLE
Snitch does not support multiline comments

### ISSUE THREAD

**Issue by notzain** [tsoding/snitch#81]
Sometimes its helpful to put comments/explanation inside a todo.

```
/*
* TODO: Fix broken function
* Currently, this function does X, because of reason ABC.
* A better solution could perhaps be Y?
*/
```

**Comment by rexim** [comment]
@notzain I added more details on how TODOs are parsed. I recommend to [check it out](https://github.com/tsoding/snitch/tree/a10dae5337d2657ef0fb5ac528adaf95298275c6#todo-format).

The problem is that snitch updates the text of TODOs and to do that it keeps track of the text before word `TODO` (aka Prefix). With multiline support it could become exponantionally difficult.

We could do the following:
- Find `TODO` and remember its prefix.
- Parse all of the consequent lines with the same prefix as the TODO's one

With this approach TODOs formated like this:

```c
/* TODO: Fix broken function
 * Currently, this function does X, because of reason ABC.
 * A better solution could perhaps be Y?
 */
```

won't work because consequent lines have a different prefix. But we could just document that as a limitation. Ideally for snitch we should use only one line comments:

```c
// TODO: Fix broken function
// Currently, this function does X, because of reason ABC.
// A better solution could perhaps be Y?
```

What do you think?

**Comment by valignatev** [comment]
How about one consecutive space? That's how IntelliJ does this
Like 
```
// TODO: title
//  Body line 1
//  Body line 2
```

**Comment by rexim** [comment]
@valignatev this approach does not give us anything since we want to be able to modify the body when needed.

**Comment by rexim** [comment]
Resolved by #122
